src/test-run/commands/index.js

```diff
@@ -57,20 +58,46 @@ function integerArgument (name, val, positive) {
 }
 
 function positiveIntegerArgument (name, val) {
-    integerArgument(name, val, true);
+    integerArgument(name, val, ActionPositiveIntegerArgumentError);
 
     if (val < 0)
         throw new ActionPositiveIntegerArgumentError(name, val);
 }
 
-function nonEmptyStringArgument (option, val) {
+function nonEmptyStringArgument (argument, val, ErrorCtor = ActionStringArgumentError, ...args) {
```

Comment:
It's getting to complicated, let's rather pass `createError` method, like @helen-dikareva did in her PR : https://github.com/helen-dikareva/testcafe-phoenix/blob/b9cdd839286957f049fc2ed3c566472ae22563b9/src/client/driver/command-executors/execute-wait-for-element-command.js#L12